.eslintrc

```diff
@@ -26,12 +26,13 @@
                 "tsx": "never"
             }
         ],
+        "no-unused-vars": "off",
+        "unused-imports/no-unused-imports": "error",
         "@typescript-eslint/ban-ts-ignore": "off",
         "@typescript-eslint/ban-ts-comment": "off",
         "prettier/prettier": "warn",
         "@typescript-eslint/interface-name-prefix": "off",
         "@typescript-eslint/no-var-requires": "warn",
```

Comment:
Kunne vi egt bare endret alle "warn" til "error"? For vi tillatter jo ikke warnings å gå gjennom

Reply:
Jo vi burde vel det, gir vel ingen mening å bruke warn?